new_project/examples/separate_apps/src/maker.ts

```diff
@@ -1,7 +1,39 @@
 import { formatEther } from "ethers/utils";
+import moment from "moment";
 import readLineSync from "readline-sync";
 import { toBitcoin } from "satoshi-bitcoin-ts";
 import { checkEnvFile, startClient } from "./lib";
+import { NegotiationProtocolHandler, Order } from "./negotiation";
+
+const defaultOrder: Order = {
+    id: "123",
+    key: "ETH-BTC",
+    valid_until: moment().unix() + 300,
+    ask: {
+        amount: "9000000000000000000",
+        asset: "ether",
+        ledger: "ethereum",
+        network: "regtest",
+    },
+    bid: {
+        amount: "100000000",
+        asset: "bitcoin",
+        ledger: "bitcoin",
+        network: "regtest",
+    },
+    execution_params: {
+        connection_info: {
+            peer_id: "UNDEFINED",
```

Comment:
🙃 I find it a bit confusing that this is set to UNDEFINED here. Similar feeling about the expiries being set to `0` below.
I understand that it is set later, but I feel when looking at the code it is a bit confusing. 
Suggestion: handle the Execution parameters outside of constructing the default order?

Reply:
1) Understood the commend regarding `UNDEFINED`, I'll just move the whole construction into the code.
2) 
> Suggestion: handle the Execution parameters outside of constructing the default order?

Can you clarify this suggestion please?